src/backend/core/api/viewsets.py

```diff
@@ -430,6 +434,30 @@ def list_waiting_participants(self, request, pk=None):  # pylint: disable=unused
         participants = lobby_service.list_waiting_participants(room.id)
         return drf_response.Response({"participants": participants})
 
+    @decorators.action(
+        detail=False,
+        methods=["POST"],
+        url_path="webhooks-livekit",
+        permission_classes=[],
+    )
+    def livekit_webhook(self, request, pk=None):  # pylint: disable=unused-argument
```

Comment:
```suggestion
    def livekit_webhook(self, request): 
```

As detail is set to False you don't need the `pk` argument